### PR TITLE
add commenting support, and add more information to notification emails

### PR DESCRIPTION
**Pull request by mspector**
@melisabok let me know if you have any suggestions!

### CODE REVIEW

File: venues/OpenReview.net/Support/init.py [hunk 1]
```diff
@@ -167,3 +167,46 @@
...
+    'signatures': ['OpenReview.net/Support'],
+    'invitees': ['everyone'],
+    'process': 'commentProcess.js',
+    'reply': {
+        'forum': None,
+        'replyto': None,
+        'invitation': 'OpenReview.net/Support/-/Request_Form',
+        'readers': {
+            'description': 'Select all user groups that should be able to read this comment.',
+            'values': ['OpenReview.net/Support']
```
Comment (melisabok) [inline venues/OpenReview.net/Support/init.py]: With this only value there is not way to reply to the contact emails.

Comment (mspector) [reply venues/OpenReview.net/Support/init.py]: Good point. I tried to make a quick fix for this, but I don't think we can get the contact email values from the request form in the comment readers field unless we create a group for them.

(in the request form itself, we can just do values-copied from the Contact Emails field)

Comment (melisabok) [reply venues/OpenReview.net/Support/init.py]: Does it work having one invitation for all the notes? May be we need to create a comment invitation per forum and add the contact emails there.

File: venues/OpenReview.net/Support/supportProcess.js [hunk 2]
```diff
@@ -2,20 +2,40 @@ function(){
...
     var programchairMailPayload = {
-      'groups': note.content['Contact Emails'],
-      'subject': 'Your request for OpenReview service has been received.',
-      'message': 'You recently requested conference management services from OpenReview. A member of our support team will contact you shortly. You can view the request here: ' + baseUrl + '/forum?id=' + note.forum
+      groups: note.content['Contact Emails'],
+      subject: 'Your request for OpenReview service has been received.',
+      message: 'You recently requested conference management services from OpenReview. A member of our support team will contact you shortly. You can view the request here: ' + baseUrl + '/forum?id=' + note.forum
     };
 
+    var commentInvitation = {
```
Comment (melisabok) [inline venues/OpenReview.net/Support/supportProcess.js]: ❤️

**Pull request merged**